src/content/en/fundamentals/security/_redirects.yaml

```diff
@@ -14,3 +14,6 @@ redirects:
 
 - from: /web/fundamentals/security/credential-management/store-credentials
   to: /web/fundamentals/security/credential-management/save-forms#store_the_credential
+
+- from: /web/fundamentals/security/encrypt-in-transit/why-https
+  to: https://web.dev/why-https-matters
```

Comment:
This isn't working on staging but I'm pretty sure it's set up correctly and will work in production